input/pagecontent/overview.md

```diff
@@ -300,7 +299,7 @@ It is up to the author to decide how to divide FSH definitions between the FSH f
 * Group things logically, for example, a profile together with its value sets, extensions, and examples.
 * Use one file for each item, and potentially divide similar items into subdirectories.
 
-A **FSH Tank** (2) refers to a directory structure, including subdirectories, that contains a single FSH project.  A FSH Tank represents a complete module that can be placed under source code control. FHIR artifacts defined elsewhere (such as profiles from another IG) are "external" and their IGs must be declared in dependencies.
+A **FSH Tank** (2) refers to a directory named **/fsh** including subdirectories, that contains `.fsh` files.  A FSH Tank represents a module that can be placed under source code control. FHIR artifacts defined elsewhere (such as profiles from another IG) are "external" and their IGs must be declared in dependencies.
```

Comment:
> A **FSH Tank** (2) refers to a directory named **/fsh**

I think this is either too specific or not specific enough; it depends on whether or not we consider project structure to be a formal part of the spec.  If we consider project structure to be an implementation detail, rather than part of the formal spec, then indicating a specific folder name is too strict.  It's probably better to leave it general (like it was before).  But if we consider it to be a part of the spec, then it's not specific enough, because really it ought to be **input/fsh**, not just **/fsh**.

> FHIR artifacts defined elsewhere (such as profiles from another IG) are "external" and their IGs must be declared in dependencies.

Based on the way you've described it above (w/ a tank being the **fsh** folder), this isn't strictly true.  In a FSH project where FSH is in **input/fsh**, other FHIR artifacts can be in **input/resources**, **input/profiles**, etc -- and they are not considered "external" nor do they have to be declared as dependencies.

I wonder if it is time to retire the tank metaphor?  It worked better when we used the **ig-data** folder inside the tank, but might not work as well now that the **fsh** folder is just one subdirectory within the whole IG.

Reply:
I agree, and I removed all references to "FSH tanks" and rewrote this section.